data/geany.glade

```diff
@@ -3498,6 +3516,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="tooltip_text" translatable="yes">Display height in rows for the autocompletion list</property>
+                                        <property name="invisible_char">●</property>
```

Comment:
I just tried sticking a checkbox in the same place and it doesn't cause any extra noise, FWIW. Using custom built Glade 3.8.5 on Ubuntu 15.10.